src/core/utils.js

```diff
@@ -1,17 +1,3 @@
 'use strict'
 
-exports.ifRepoExists = (repo, cb) => {
-  repo.exists((err, exists) => {
-    if (err) {
-      return cb(err)
-    }
-
-    if (exists === false) {
-      return cb(new Error('no ipfs repo found'))
-    }
-
-    cb()
-  })
-}
-
```

Comment:
I didn't know this was in the readme, this is not needed anymore internally that's why I removed it